packages/cozy-harvest-lib/src/components/AccountForm/AccountField.jsx

```diff
@@ -52,7 +53,7 @@ export class AccountField extends PureComponent {
     const isEditable = !(role === ROLE_IDENTIFIER && initialValue)
 
     // Cozy-UI <Field /> props
-    const fieldProps = {
```

Comment:
Why should we do that? Falsy props should correctly be handled :/

Reply:
I'll remove this, and actually count on Cozy-UI to not stylize component with `aria-disabled=false`. See https://github.com/cozy/cozy-ui/pull/894

Reply:
Let's use the last cozy-ui and remove this so 👍